Core/Graphics/Source/Apple/macOS/GraphicsPlatform.mm

```diff
@@ -0,0 +1,27 @@
+#include <Babylon/Graphics.h>
+#include <Babylon/GraphicsPlatform.h>
+#include "../GraphicsImpl.h"
+
+#include <AppKit/Appkit.h>
+#include <MetalKit/MetalKit.h>
+
+namespace Babylon
+{
+
+    void GraphicsImpl::ConfigureBgfxPlatformData(const GraphicsConfiguration& config, bgfx::PlatformData& pd)
+    {
+        pd.ndt = nullptr;
+        pd.nwh = config.windowPtr;
+        pd.context = nullptr;
+        pd.backBuffer = nullptr;
+        pd.backBufferDS = nullptr;
+    }
+
+    float GraphicsImpl::UpdateDevicePixelRatio()
+    {
+        std::scoped_lock lock{m_state.Mutex};
+        MTKView* view = (MTKView*)GetNativeWindow<WindowType>();
```

Comment:
If this has to be cast, then haven't we lost the benefit of the new approach?

Reply:
Good point, I think that was an oversight, I don;t believe I've updated this code in a while.

Reply:
removed the cast